src/components/VideoCard/VideoCard.Styled.jsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
+export const VideoCardContainer = styled.div`
+  width: 345px;
+  height: 345px;
+  margin: 10px;
+  overflow: hidden;
+  box-shadow: 0px 1px 3px 0px rgba(255, 255, 255, 0.88);
+  border-radius: 4px;
+  color: #ececec;
```

Comment:
I recommend handling the colors with a theme provider, so changing colors with the light and dark theme becomes easier:

https://styled-components.com/docs/advanced#function-themes